src/modules/aws_codebuild/rpcs/import_source_credential.ts

```diff
@@ -0,0 +1,82 @@
+import { Build, CodeBuild, ImportSourceCredentialsInput } from '@aws-sdk/client-codebuild';
+import { ServerType } from '@aws-sdk/client-codebuild/dist-types/models/models_0';
+
+import { AWS, crudBuilderFormat } from '../../../services/aws_macros';
+import { Context, RpcBase, RpcResponseObject } from '../../interfaces';
+import { AwsCodebuildModule } from '../index';
+
+export class ImportSourceCredentialRpc extends RpcBase {
```

Comment:
can you add typedoc documentation as i did for other rpcs? so we keep the documentation up to date

Reply:
Added.